src/Stripe.net/Services/Persons/PersonSharedOptions.cs

```diff
@@ -65,6 +64,7 @@ public abstract class PersonSharedOptions : BaseOptions, IHasMetadata
         public PersonRelationshipOptions Relationship { get; set; }
 
         [JsonProperty("ssn_last_4")]
+        [AllowNameMismatch]
         public string SSNLast4 { get; set; }
```

Comment:
Should this have been `SsnLast4` instead?

Reply:
Probably, though that's not the reason for the mismatch: both `SsnLast4` and `SSNLast4` become `ssn_last4` instead of `ssn_last_4` when converted to snake_case.

Reply:
ah yeah makes sense, let's leave it in that case